src/services/dataContainerService.ts

```diff
@@ -117,6 +117,86 @@ export class ContainerService {
       );
   }
 
+  /**
```

Comment:
@tristantarrant which templates should be propose from the console for this version (12) and remove the ones from the server for create cache

Reply:
I put that in the code like that but I will probably move this temporal stuff to another file. I want to validate configurations we want to propose for this, and don't any further change in the code.

Reply:
hey, as mentioned in Zulip it seems like the best reason for providing a cache template in the console is to get users off the ground. rather than flood with choices, make it simple and provide one option that works for the getting started use case.

what about keeping only "Distributed Protobuf Encoded Cache" and renaming it to something like "Quick start cache"?

Reply:
Good idea @oraNod it can be the replacement for the protobuf example template with no doubt. let's see what @tristantarrant says. I just don't want to confuse users anymore with useless templates. the day we will remove those and deal templates better, we might want to provide a list. but this is work for 13 (I haven't started the fork between 12/13 from the console yet)